### PR TITLE
Scan aglossry.tex until it converges

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

**Comment by womullan** [comment]
@jdswinbank this is not the niceest way to do it but certainly easiest and removes the uncertain number of calls in the makefile.

**Comment by jdswinbank** [comment]
Nice – will try this out shortly.

File: bin/generateAcronyms.py [hunk 1]
```diff
@@ -429,6 +428,24 @@ def write_latex_table(acronyms, fd=sys.stdout):
     print(r"\end{longtable}", file=fd)
 
 
+def forceConverge(incount, doGlossary, utags):
+    """Run throug the glossary looking for defnitions untill
```
Comment (jdswinbank) [inline bin/generateAcronyms.py]: "through".

File: bin/generateAcronyms.py [hunk 2]
```diff
@@ -429,6 +428,24 @@ def write_latex_table(acronyms, fd=sys.stdout):
     print(r"\end{longtable}", file=fd)
 
 
+def forceConverge(incount, doGlossary, utags):
```
Comment (jdswinbank) [inline bin/generateAcronyms.py]: I'm not sure what you gain by passing `doGlossary` here — if you are calling `forceConverge` then `doGlossary` *must* be true, or you have already lost. I would pull this check outside the function.

File: bin/generateAcronyms.py [hunk 3]
```diff
@@ -429,6 +428,24 @@ def write_latex_table(acronyms, fd=sys.stdout):
...
+    """Run throug the glossary looking for defnitions untill
+    no more are added.
+    """
+    if (not doGlossary):
```
Comment (jdswinbank) [inline bin/generateAcronyms.py]: Here and below, there are a lot more parens than you actually need.

**Comment by jdswinbank** [comment]
PR: https://github.com/lsst/lsst-texmf/pull/190

File: bin/generateAcronyms.py (hunk 3 above)
Comment (womullan) [reply bin/generateAcronyms.py]: since i started with LISP .. I feel i always am short of parens ..

**Pull request merged**